Week-3/exercise_2/index.js

```diff
@@ -0,0 +1,73 @@
+/*
+ Bind function example
+ 1. .bind is permanently bind 
+ 2. The bind() method creates a new function when called, has its this keyword set to provided value with given sequence of arguments preceding any provided when the new function is called.
+*/
+
+function multiply(a, b){
+    return this.a * this.b;
```

Comment:
Cant it be achieved without the use of `this` keyword?

Reply:
No , because  The bind() method creates a new function when called, has its this keyword set to provided value with given sequence of arguments preceding any provided when the new function is called